c++/src/Reader.cc

```diff
@@ -993,12 +995,25 @@ namespace orc {
     }
   }
 
+  int64_t getEpochOffset() {
+    struct tm epoch;
+    epoch.tm_sec = 0;
+    epoch.tm_min = 0;
+    epoch.tm_hour = 0;
+    epoch.tm_mday = 1;
+    epoch.tm_mon = 0;
+    epoch.tm_year = 2015 - 1900;
+    epoch.tm_isdst = 0;
+    return static_cast<int64_t>(mktime(&epoch));
+  }
+
```

Comment:
I'm a bit confused: Why is the offset 115 years and 1 day, instead of 45 years (2015 - 1970)?